docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -105,105 +112,95 @@ significant changes in each minor release.
 
-Changelogs
-^^^^^^^^^^^^
-
-We generate changelogs based on fragments. Here is the generated changelog for 2.9_ as an example. When creating new features or fixing bugs, create a changelog fragment describing the change. A changelog entry is not needed for new modules or plugins. Details for those items will be generated from the module documentation.
+If there are major problems with the first candidate, the team and the community fix them and tag a second release candidate (RC2). This second candidate lasts for a shorter duration than the first. If no problems have been reported for an RC2 after two business days, the second release candidate becomes the final release.
 
-We've got :ref:`examples and instructions on creating changelog fragments <changelogs_how_to>` in the Community Guide.
+If there are major problems in RC2, the cycle begins again with another release candidate and repeats until the core maintainers agree that all major problems have been fixed.
 
 
-Release candidates
-^^^^^^^^^^^^^^^^^^^
+.. _development_and_stable_version_maintenance_workflow:
 
-Before a new release or version of Ansible or ``ansible-core`` can be
-done, it will typically go through a release candidate process.
+Development and stable version maintenance workflow
+----------------------------------------------------
 
-This provides the Ansible community the opportunity to test these releases and report
-bugs or issues they might come across.
+In between releases, the Ansible community develops and maintains ``ansible-core`` on GitHub_, with a workflow that looks like this:
 
-Ansible and ``ansible-core`` tag the first release candidate (``RC1``)
-which is usually scheduled to last five business days. The final release
-is done if no major bugs or issues are identified during this period.
+ * Developers add new features and bug fixes to the ``devel`` branch.
+ * Each new feature and each bug fix includes a changelog fragment describing the work.
+ * The development team backports bug fixes to one, two, or three stable branches, depending on the severity of the bug. They do not backport new features.
+ * Release engineers create a minor release for each maintained version every three weeks to ensure that the latest bug fixes are available to users.
+ * At the end of the development period, the release engineers impose a feature freeze and the work of creating a new release begins.
 
-If there are major problems with the first candidate, a second candidate will
-be tagged (``RC2``) once the necessary fixes have landed.
-This second candidate lasts for a shorter duration than the first.
-If no problems have been reported after two business days, the final release is
-done.
+We generally do not provide fixes for unmaintained releases of Ansible, however, there can sometimes be exceptions for critical issues.
```

Comment:
Is it worth adding how the Ansible project develops, since the prior list seems focused on ansible-core? Maybe something as simple as 'The Ansible community package is developed in multiple repositories that source each collection. As such, there is no single  ``devel`` branch.

Then we can elaborate on that once we have a devel docsite for Ansible to say what it's reflecting.

Reply:
Yep, good idea. I split this section up so we have Workflow for core and Workflow for the package.